api/src/handlers/getPosts/getPostsResponse.js

```diff
@@ -0,0 +1,19 @@
+const DATABASE_ERROR_MESSAGE = "Couldn't get posts from database.";
+const CORS_HEADER = {'Access-Control-Allow-Origin': '*'};
+
+module.exports = async function getPostsResponse(db) {
+  try {
+    const body = await db.getAuthorPosts();
+    return {
+      statusCode: 200,
+      headers: CORS_HEADER,
+      body: JSON.stringify(body),
+    };
+  } catch (err) {
```

Comment:
Is that only returned to the user? It feels like it could be logged as well if you want to be able to monitor the application